backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/slider/internal/BpkRangeSliderImpl.kt

```diff
@@ -60,6 +60,10 @@ internal fun BpkRangeSliderImpl(
 ) {
     val startInteractionSource = remember { MutableInteractionSource() }
     val endInteractionSource = remember { MutableInteractionSource() }
+
+    val isLowerThumbBeingTouched = startInteractionSource.collectIsDraggedAsState().value
```

Comment:
NIT: collectIsDraggedAsState() will not react on being touched, it will react on being moved. Perhabs you want to use collectIsPressedAsState().

Reply:
actually I tried using `collectIsPressedAsState ()` first but it didn't work, that's why I used  `collectIsDraggedAsState()`